nussknacker-dist/src/universal/conf/application.conf

```diff
@@ -18,25 +19,50 @@ scenarioTypes {
         enable: ${?FLINK_ROCKSDB_ENABLE}
       }
       components.kafka {
-        categories: ["Default"]
-        config: {
-          kafkaAddress: ${?KAFKA_ADDRESS}
-          kafkaProperties {
-            "schema.registry.url": ${?SCHEMA_REGISTRY_URL}
-          }
-        }
+        config: ${kafkaConfig}
+      }
+    }
+  }
+  "streaming-lite-embedded": {
+    deploymentConfig: ${streamingLiteDeploymentConfig}
+    modelConfig: {
+      classPath: ["model/defaultModel.jar", "components/lite/liteBase.jar", "components/lite/liteKafka.jar", "components/common"]
+      classPath: ${?MODEL_CLASS_PATH}
+
+      kafka: ${kafkaConfig}
+      exceptionHandlingConfig: {
+        topic: "errors"
       }
     }
   }
 }
 
 flinkDeploymentConfig {
-  jobManagerTimeout: 1m
   type: "flinkStreaming"
+  jobManagerTimeout: 1m
   restUrl: ${?FLINK_REST_URL}
   queryableStateProxyUrl: ${?FLINK_QUERYABLE_STATE_PROXY_URL}
 }
 
+streamingLiteDeploymentConfig {
```

Comment:
Is there a reason to put it here?

Reply:
Just to be consistent with `flinkDeploymentConfig`. Should I inline both? WDYT?

Reply:
I'd inline both, there is some reason to extract it in dev conf, as we share config between many configs, but here it's just noise IMO